src/main/scala/nl/knaw/dans/easy/multideposit/actions/AddDatasetMetadataToDeposit.scala

```diff
@@ -115,9 +115,7 @@ object AddDatasetMetadataToDeposit {
       <dcx-dai:contributorDetails>{
         if (isOrganization(mdKeyValues))
         <dcx-dai:organization>
-          <dcx-dai:name xml:lang="en">
-            {mdKeyValues.find(field => organizationKeys.contains(field._1)).map(_._2).getOrElse("")}
-          </dcx-dai:name>
+          <dcx-dai:name xml:lang="en">{mdKeyValues.find(field => organizationKeys.contains(field._1)).map(_._2).getOrElse("")}</dcx-dai:name>
```

Comment:
If the opening brace doesn't hug the starting tag, we'll get extra space before the value.